config/routes.rb

```diff
@@ -29,8 +29,8 @@
           get "handout"
           post "submit"
           post "set_group_settings"
-
-          resources :groups, only: [:index, :create, :destroy]
+          
```

Comment:
Nit: remove the whitespace introduced here

Reply:
Done!